### PR TITLE
Switch the registration notification email to be a list in settings.

### PR DESCRIPTION
**Pull request by FlipperPA**

### CODE REVIEW

File: wagtailspace/settings/base.py [hunk 1]
```diff
@@ -130,3 +130,8 @@
 }
 
 CRISPY_TEMPLATE_PACK = 'bootstrap4'
+
+WAGTAILSPACE_REGISTRATION_NOTIFICATIONS = [
```
Comment (willbarton) [inline wagtailspace/settings/base.py:134]: Why not set this based on an environment variable, then we can configure it in Heroku (or wherever else it's hosted)?

Comment (FlipperPA) [reply wagtailspace/settings/base.py:134]: Good call. This should do the trick, and return an empty list if the environment variable isn't populated, guaranteeing the setting will always be a list.

Comment (tomdyson) [reply wagtailspace/settings/base.py:134]: LGTM!

Comment (willbarton) [reply wagtailspace/settings/base.py:134]: That works for me!

**Pull request merged**